central/credentialexpiry/service/service_impl.go

```diff
@@ -46,6 +49,15 @@ func (s *serviceImpl) GetCertExpiry(ctx context.Context, request *v1.GetCertExpi
 		return s.getCentralCertExpiry()
 	case v1.GetCertExpiry_SCANNER:
 		return s.getScannerCertExpiry(ctx)
+	// TODO(ROX-20064): Needs to be implemented when Scanner V4 has been added to imageIntegrations.
```

Comment:
I don't understands What exactly do you mean by "when Scanner V4 has been added to imageIntegrations"?

Isn't scannerV4 just another Integration of type ImageIntegration_ClairV4?

If so this should already be working or we only need to add the SN for scanner v4 to the matcher [here](https://github.com/stackrox/stackrox/blob/74e9d50a8fa8cf53bd3fe04438488828a1d0834f/central/credentialexpiry/service/service_impl.go#L103)

Reply:
Thats right, ScannerV4 and ClairV4 are different.

Both use the `claircore` engine - but are different wrappers on top of the engine.  Features such as 'node scanning' will eventually be integrated into ScannerV4 but not ClairV4 (for example). We will be able to add ACS specific functionality into ScannerV4, something that we cannot do with ClairV4.

https://github.com/stackrox/stackrox/pull/8869 introduces the ScannerV4 integration (pending approval/review)

Reply:
Thanks for explaining.